ql/src/java/org/apache/hadoop/hive/ql/metadata/Hive.java

```diff
@@ -905,6 +921,7 @@ public void alterPartition(String catName, String dbName, String tblName, Partit
       throw new HiveException("Unable to alter partition. " + e.getMessage(), e);
     } catch (TException e) {
       throw new HiveException("Unable to alter partition. " + e.getMessage(), e);
+    } finally {
```

Comment:
Empty finally?

Reply:
Rebased with master and only kept relevant changes.